geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/tools/mapreduce/JobContextConfigurationWrapper.java

```diff
@@ -3,16 +3,15 @@
 import mil.nga.giat.geowave.accumulo.mapreduce.GeoWaveConfiguratorBase;
 import mil.nga.giat.geowave.analytics.tools.ConfigurationWrapper;
 import mil.nga.giat.geowave.index.ByteArrayUtils;
-import mil.nga.giat.geowave.index.PersistenceUtils;
-import mil.nga.giat.geowave.store.adapter.DataAdapter;
 
 import org.apache.hadoop.mapreduce.JobContext;
-import org.apache.log4j.Logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class JobContextConfigurationWrapper implements
 		ConfigurationWrapper
 {
-	protected static final Logger LOGGER = Logger.getLogger(JobContextConfigurationWrapper.class);
+	protected static final Logger LOGGER = LoggerFactory.getLogger(JobContextConfigurationWrapper.class);
 
 	private final JobContext context;
 	private Logger logger = LOGGER;
```

Comment:
Wouldn't let me comment on it, but just below this
```
	public JobContextConfigurationWrapper(
			final JobContext context,
			final Logger logger ) 
```

What's the rationale here for passing around a logger instance?